### DISCUSSION TITLE
Issue with PnP applied filters - Existed filters also applying for new search keyword.

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2533] [Q&A]
Hi Team,

I'm facing issue with PnP Filters were already applied filters are applying for new search keyword. 
Ex: At first time search with keyword, Post search results applied the filters and then researched with another search keyword there also filters are applying and creating a frustration to the users.  
Checked SharePoint classic search and Office.com but this scenario was not existed? 

Can anyone help me on how to fix this filter issue with when re-searching with new keyword or moving between the vertices?

**Comment by wobba** [comment]
There is no option for this today in v4, and follows the filter behavior of modern search where filters stick across sessions until the user clears it.
